commands/template_pull_test.go

```diff
@@ -124,3 +127,72 @@ func Test_repositoryUrlRemoteRegExp(t *testing.T) {
 		})
 	}
 }
+
+// templatePullLocalTemplateRepo executes `template pull` on a local repository to get templates
+func templatePullLocalTemplateRepo(t *testing.T) {
```

Comment:
What's this additional code doing with zips? I thought the changes @LucasRoesler made removed the need for zips now?

Reply:
I have a zip containing the templates repo, that zip is to used to have a local repo for tests.

`commands/testdata/templates.git.zip`

Reply:
replace with git submodule & git init; git add; git commit to create a local repo during the tests